wagtail/images/templates/wagtailimages/edit_handlers/multiple_images_panel.html

```diff
@@ -0,0 +1,72 @@
+{% extends "wagtailadmin/edit_handlers/inline_panel.html" %}
```

Comment:
The duplication of code here with [`wagtail/images/templates/wagtailimages/multiple/add.html`](https://github.com/wagtail/wagtail/blob/8e0b2f6d1da6c18ad1aaaa8366f02e6c8cf76027/wagtail/images/templates/wagtailimages/multiple/add.html) concerns me a bit. Is there no way to refactor this to share code somehow? Otherwise we risk one uploader getting fixes/improvements but not the other.